ports/cef/window.rs

```diff
@@ -104,69 +104,68 @@ impl Window {
-            Cursor::Crosshair => return cef_cursor_type_t::CT_CROSS,
-            Cursor::Copy => return cef_cursor_type_t::CT_COPY,
-            Cursor::Alias => return cef_cursor_type_t::CT_ALIAS,
-            Cursor::Text => return cef_cursor_type_t::CT_IBEAM,
-            Cursor::Grab | Cursor::AllScroll =>
-                return cef_cursor_type_t::CT_GRAB,
-            Cursor::NoDrop => return cef_cursor_type_t::CT_NODROP,
-            Cursor::NotAllowed => return cef_cursor_type_t::CT_NOTALLOWED,
-            Cursor::Pointer => return cef_cursor_type_t::CT_POINTER,
-            Cursor::SResize => return cef_cursor_type_t::CT_SOUTHRESIZE,
-            Cursor::WResize => return cef_cursor_type_t::CT_WESTRESIZE,
-            Cursor::EwResize => return cef_cursor_type_t::CT_EASTWESTRESIZE,
-            Cursor::ColResize => return cef_cursor_type_t::CT_COLUMNRESIZE,
-            Cursor::EResize => return cef_cursor_type_t::CT_EASTRESIZE,
-            Cursor::NResize => return cef_cursor_type_t::CT_NORTHRESIZE,
-            Cursor::NsResize => return cef_cursor_type_t::CT_NORTHSOUTHRESIZE,
-            Cursor::RowResize => return cef_cursor_type_t::CT_ROWRESIZE,
-            Cursor::VerticalText => return cef_cursor_type_t::CT_VERTICALTEXT,
-            _ => return cef_cursor_type_t::CT_POINTER,
+            CursorKind::None => cef_cursor_type_t::CT_NONE,
+            CursorKind::ContextMenu => cef_cursor_type_t::CT_CONTEXTMENU,
+            CursorKind::Grabbing => cef_cursor_type_t::CT_GRABBING,
+            CursorKind::Crosshair => cef_cursor_type_t::CT_CROSS,
+            CursorKind::Copy => cef_cursor_type_t::CT_COPY,
+            CursorKind::Alias => cef_cursor_type_t::CT_ALIAS,
+            CursorKind::Text => cef_cursor_type_t::CT_IBEAM,
+            CursorKind::Grab | CursorKind::AllScroll => cef_cursor_type_t::CT_GRAB,
+            CursorKind::NoDrop => cef_cursor_type_t::CT_NODROP,
+            CursorKind::NotAllowed => cef_cursor_type_t::CT_NOTALLOWED,
+            CursorKind::Pointer => cef_cursor_type_t::CT_POINTER,
+            CursorKind::SResize => cef_cursor_type_t::CT_SOUTHRESIZE,
+            CursorKind::WResize => cef_cursor_type_t::CT_WESTRESIZE,
+            CursorKind::EwResize => cef_cursor_type_t::CT_EASTWESTRESIZE,
+            CursorKind::ColResize => cef_cursor_type_t::CT_COLUMNRESIZE,
+            CursorKind::EResize => cef_cursor_type_t::CT_EASTRESIZE,
+            CursorKind::NResize => cef_cursor_type_t::CT_NORTHRESIZE,
+            CursorKind::NsResize => cef_cursor_type_t::CT_NORTHSOUTHRESIZE,
+            CursorKind::RowResize => cef_cursor_type_t::CT_ROWRESIZE,
+            CursorKind::VerticalText => cef_cursor_type_t::CT_VERTICALTEXT,
+            _ => cef_cursor_type_t::CT_POINTER,
         }
     }
 
     /// Returns the Cocoa cursor for a CSS cursor. These match Firefox, except where Firefox
     /// bundles custom resources (which we don't yet do).
     #[cfg(target_os="macos")]
```

Comment:
Yeah, this function is macos only, so if you're on Linux like I am you don't even build it.